reference/channels.v3.yml

```
@@ -1631,6 +1738,15 @@ components:
       format: int64
       description: The ID of the channel.
       x-internal: false
+    PutCheckoutUrl:
+      type: object
+      properties:
+        url:
+          type: string
+          description: Unique site checkout URL. Must be valid and a sub-domain of primary channel url.
```

```suggestion
          description: Unique site checkout URL. Must be valid and a sub-domain of primary channel URL.
```
find and replace url --> URL where appropriate.